### DISCUSSION TITLE
InfluxDB 2.x support

### DISCUSSION THREAD

**Discussion by codicusmaximus** [hassio-addons/addon-influxdb#353] [General]
Is there a plan to eventually migrate to InfluxDB 2.x for this add-on?

I have a need for the "Edge Data Replication" feature introduced in InfluxDB 2.2 since I want my local home assistant's InfluxDB data to be replicated to a cloud backup InfluxDB instance but since this plugin is on 1.x I cannot yet make this happen.

I do see that [this](https://community.home-assistant.io/t/home-assistant-add-on-influxdb-v2/617977) exists and I may try it; however ideally I would like to stick with this existing project/addon if possible.  Just want to know if InfluxDB v2 is considered or on some kind of future vision or roadmap etc.

**Comment by StevenSlaa** [comment]
I have created a pull-request for this: #356

**Comment by schmidtfx** [comment]
Is there any progress on this?
